ironmon_tracker/constants/MoveData.lua

```diff
@@ -2425,7 +2425,7 @@ MoveData.MOVES_MASTER_LIST = {
 		pp = "5",
 		accuracy = "100",
 		category = MoveData.MOVE_CATEGORIES.STATUS,
-		description = "The next 4-8 times the target attempts to move, it is forced to repeat its last used move."
+		description = {"The next 3-7(DP)/2-6(Pt)/4-8(HGSS) times the target attempts to move, it is forced to repeat its last used move.", "The next 3-7(DP)/2-6(Pt)/4-8(HGSS) times the target attempts to move, it is forced to repeat its last used move.", "The next 3-7(DP)/2-6(Pt)/4-8(HGSS) times the target attempts to move, it is forced to repeat its last used move.", "The next 3-7(DP)/2-6(Pt)/4-8(HGSS) times the target attempts to move, it is forced to repeat its last used move.", "The next 3 times the target attempts to move, it is forced to repeat its last used move."}
```

Comment:
To keep consistency with the way parenthesized items are drawn on the tracker, these should have a space before (and possibly after)

Reply:
added space before and after